sql/sqlserver/server.go

```diff
@@ -284,6 +284,292 @@ func (s *Server) Use(p *parser.Use, args []sqlwire.Datum, resp *sqlwire.Response
 	s.database = p.Name
```

Comment:
I didn't notice this in the previous commit, but this isn't correct. Add a TODO that setting of the database name needs to set a field in the session settings of the reply. The `Server` object itself needs to be stateless.